keg_bouncer/db/__init__.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import absolute_import
+
+import sqlalchemy as sa
+import sqlalchemy.orm as saorm
+from keg.db import db
+from keg_elements.db.mixins import MethodsMixin
+
+
+class Permission(db.Model, MethodsMixin):
```

Comment:
Normally we put entities in `project/model/entities`. While Keg itself uses a db folder and so does Keg-Elements, I think that is because those are for helper classes. @rsyring can make a final call, he might want to switch to a `project/db/entities` or something along those lines. Regardless, I don't think `__init__` is the best place for this.